CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 - DNA Tester works now with more than one fingerprint on a weapon
 - TraitorButton config files should now actually work
 - Translation strings not rendering on detective's body search mode combobox
+- Attempting to unscope from weapons without ironsights
```

Comment:
Same here, should be:
```suggestion
- Disable to unscope from weapons without ironsights
```

Reply:
probably "prevent" is the better word here